{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -372,9 +380,10 @@ def import_media(c):
     Rsync the distant media folder content, to the local media folder (identified by
     the MEDIA_ROOT environment variable).
     """
-    if "MEDIA_ROOT" not in os.environ:
+    local_media_root = get_local_env_variable("MEDIA_ROOT")
+    if not local_media_root:
```

Comment:
If MEDIA_ROOT is not in `os.environ` and not in the `envdir` path, a `FileNotFoundError` will be raised. Maybe handle the exception here or in the `get_loca_env_variable` function?